tests/Doctrine/Migrations/Tests/Tools/Console/Command/DiffCommandTest.php

```diff
@@ -55,9 +55,14 @@ public function testExecute() : void
         $input->expects(self::at(4))
             ->method('getOption')
             ->with('check-database-platform')
-            ->willReturn(true);
+            ->willReturn(false);
```

Comment:
This change is just to have a value different from the default (like all other tested options) and for the call below